evaluation-tasks/README.md

```diff
@@ -59,3 +59,19 @@ If the dataset provides a validation.csv, that will be included
 too. Otherwise, participants do partition train into train/val
 however they like.
 
+## Caching with S3
+
+1. Download and configure the AWS CLI if you haven't done that already:
+    * [Intallation](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+    * [Configuration](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html)
+
+2. Update the S3 config file: `config/s3.py`
+    * `S3_CACHE = True` enables S3 caching. Set this to False if you want to disable
+        caching for all tasks.
```

Comment:
What does caching mean here?

Reply:
This basically just turns on and off the S3 caching. Won't need this if we pull out the S3 stuff into a separate script.